### PR TITLE
Feat/componenttests

### PR DESCRIPTION
**Pull request by PhilippWeiss85**
Wrote some simple component rendering tests. Please check. Thanks!

### CODE REVIEW

File: components/GameCard/GameCard.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        time="20:00"
+        place="Fideliopark"
+        court="Gras"
+        id="1"
+        results={testingResults}
+      />
+    );
+
+    expect(screen.getByText("Fideliopark")).toBeVisible();
+    expect(screen.getAllByText("Gras")).toHaveLength(1);
```
Comment (F-Kirchhoff) [inline components/GameCard/GameCard.test.js:33]: this test doesn't check if all elements are displayed correctly. Think about all elements that are generated dynamically and test each of them.

File: components/GameDetailForm/GameDetailForm.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    expect(screen.getByText("Set 1")).toBeVisible();
+    expect(screen.getByText("Set 2")).toBeVisible();
+    expect(screen.getByText("Set 3")).toBeVisible();
+  });
+
+  it("result numbers should be a viable type", () => {
+    render(<GameDetails id="3" showMoreDetails="true" results={testingResults} />);
+
+    expect(screen.getAllByRole("numberinput")).toBeTruthy();
+    expect(screen.getAllByRole("numberinput")).not.toBeFalsy();
```
Comment (F-Kirchhoff) [inline components/GameDetailForm/GameDetailForm.test.js]: these two tests do exactly the same, one is enough.

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey Phillip, very good that you write tests for your components. Please make sure that you
test all dynamic elements of your component an not just some parts of it.

File: components/GameCard/GameCard.test.js (hunk 1 above)
Comment (PhilippWeiss85) [reply components/GameCard/GameCard.test.js:33]: Added checks for every element in the card.

File: components/GameDetailForm/GameDetailForm.test.js (hunk 2 above)
Comment (PhilippWeiss85) [reply components/GameDetailForm/GameDetailForm.test.js]: removed toBeFalsy()

**Review by F-Kirchhoff — APPROVED** [review]
LGTM 🦊

**Pull request merged**